conf/defaults.ini

```diff
@@ -1299,6 +1299,17 @@ loki_basic_auth_password =
 # ex.
 # mylabelkey = mylabelvalue
 
+[unified_alerting.state_history.annotations]
+# Controls retention of annotations automatically created while evaluating alert rules
+# when alerting state history backend is configured to be annotations (see setting [unified_alerting.state_history].backend).
```

Comment:
```suggestion
# Controls retention of annotations automatically created while evaluating alert rules. 
# Alert state history backend must be configured to be annotations (see setting [unified_alerting.state_history].backend).
```

Reply:
"must be configured to be annotations" sounds odd - what do you mean? Annotations must be configured in the alert state history backend? The alert state history backend must have annotations configured.?

Reply:
alert state history has two backend modes: loki and annotations API. The new section only takes effect when backend is "annotations"